src/lib/datepicker/datepicker.spec.ts

```diff
@@ -164,12 +164,16 @@ describe('MdDatepicker', () => {
         testComponent.datepicker.close();
         fixture.detectChanges();
 
-        fixture.whenStable().then(() => {
-          expect(document.querySelector('md-dialog-container')).toBeNull();
-        });
-      });
+        tick();
 
-      it('setting selected should update input and close calendar', () => {
+        expect(document.querySelector('md-dialog-container')).toBeNull();
+
+        // The fake async zone expects all active timers to be done at the end of the test.
```

Comment:
I'm not sure that this kind of comment is necessary for every `tick` call.

Reply:
Yeah totally see what you mean. It's just that it can be confusing if we place a `tick(500)` after every test.